src/js/modules/infragistics.ui.editors.js

```diff
@@ -12059,5 +12062,695 @@
 			}
 		}
 	});
+	$.widget("ui.igTimePicker", $.ui.igDateEditor, {
+		options: {
+			/* type="object" Gets/Sets delta-value which is used to generate the drop-down items for the time picker.
+				Accepted values for deltas are positive integer numbers, and the fractional portion of floating point numbers is ignored.
+			```
+				//Initialize with object
+				$(".selector").%%WidgetName%%({
+					itemsDelta: {
+						hours: 0,
+						minutes: 30,
+					}
+				});
+
+				//Get
+				var itemsDelta= $(".selector").%%WidgetName%%("option", "itemsDelta");
+
+				//Set with object
+				$(".selector").%%WidgetName%%("option", "itemsDelta", { minutes: 30 });
+			```
+			object type="object" A configuration object, which defines specific values for each time period. The option can accept the following format:
+				itemsDelta: {
+					hours: 0,
+					minutes: 30,
+				}
+			Time periods that don't have values use 0 as default for hours and 30 for minutes.
+			*/
+			itemsDelta: { hours: 0, minutes: 30 },
+			/* type="number|object" Gets/Sets delta-value which is used to increment or decrement the editor date on spin actions.
+				When not editing (focused) the delta is applied on the day if available in the input mask or the lowest available period.

```

Comment:
No chance of delta being applied to day, right?

Reply:
yes